pkg/features/kube_features.go

```diff
@@ -702,7 +703,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	SupportNodePidsLimit:                           {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.23
 	BoundServiceAccountTokenVolume:                 {Default: true, PreRelease: featuregate.Beta},
 	ServiceAccountIssuerDiscovery:                  {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.22
-	CRIContainerLogRotation:                        {Default: true, PreRelease: featuregate.Beta},
+	CRIContainerLogRotation:                        {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.22
```

Comment:
Can you please remove any conditionals that rely on the feature gate? Namely,

https://github.com/kubernetes/kubernetes/blob/a5c3b24dba3975bd09e37e5266e9465bbf76bcf9/pkg/kubelet/kubelet.go#L602

I think this test also needs an update:

https://github.com/kubernetes/kubernetes/blob/a5c3b24dba3975bd09e37e5266e9465bbf76bcf9/test/e2e_node/container_log_rotation_test.go#L53-L57

Reply:
yup, fixed